src/dataflow/src/server.rs

```diff
@@ -504,6 +500,11 @@ where
             }
         }
     }
+    fn update_rt_timestamps(&self) {
```

Comment:
Toss some doc comments here please. What does this do, who should call it, why, etc.

Reply:
done